src/components/Offramp/PromoCodeChecker.tsx

```
@@ -0,0 +1,117 @@
+                isApplied: false,
+            }))
+            onPromoCodeApplied(null)
+        } else {
+            {
+                setPromoCheckerState((prev) => ({
+                    ...prev,
+                    error: 'Invalid promo code',
+                    isApplied: false,
+                }))
+                onPromoCodeApplied(null)
+            }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)